src/main/java/sc/fiji/pQCT/analysis/DistributionAnalysis.java

```diff
@@ -147,10 +147,10 @@ public DistributionAnalysis(final SelectROI roi,
 		*/
 		maxRadius = range(0, peeledSize).filter(i -> peeledROI[i] >= threshold)
 			.mapToDouble(index -> {
-				final int j = index / width;
+				final int j = (int) Math.floor(index / width);
 				final int i = index - width * j;
-				final double x = i - marrowCenter[0];
-				final double y = j - marrowCenter[1];
+				final double x = ((double) i) - marrowCenter[0];
+				final double y = ((double) j) - marrowCenter[1];
 				//IJ.log("X "+x+" Y "+y+" mx "+x+" my "+y+" rad "+Math.sqrt(x * x + y * y));
```

Comment:
Doubles implicitly cast in assignment, `(double)` not needed.

Reply:
Also, if any variable in a calculation is double, all the other variables are implicitly cast to double.

Reply:
Why are variables not `final` anymore?